images/prow-tests-go114/Dockerfile

```diff
@@ -15,10 +15,25 @@
 FROM gcr.io/knative-tests/test-infra/prow-tests:stable
 LABEL maintainer "Chao Dai <chaodai@google.com>"
 
-# Install go at 1.14 same as how it's installed in kubekins-e2e image, reference code here:
-# https://github.com/kubernetes/test-infra/blob/1e9b5dc3de4b268aab57c9c48120aa3dcf096bc6/images/kubekins-e2e/Dockerfile#L64
-ENV GO_TARBALL "go1.14.linux-amd64.tar.gz"
-RUN rm -rf /usr/local/go && \
-    wget -q "https://storage.googleapis.com/golang/${GO_TARBALL}" && \
+ENV GO_TARBALL "go1.14beta1.tar.gz"
+
+# Required variables to install go from source
+ENV GOROOT_BOOTSTRAP "/usr/local/go-old"
+ENV GOROOT "/usr/local/go"
+ENV CGO_ENABLED 0
+ENV GOOS "linux"
+ENV GOARCH "amd64"
+
+# install netbase as it's required for Go linux amd64 install (OS specific)
+RUN apt-get update && \
```

Comment:
This was already ran https://github.com/knative/test-infra/blob/77b975af074880c49957946a6d861e063e11f232/images/prow-tests/Dockerfile#L25